finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -15,6 +17,17 @@ private[netty4] class ServerBridge[In, Out](
 
   def initChannel(ch: Channel): Unit = {
     val transport: Transport[In, Out] = transportFac(ch)
-    serveTransport(transport)
```

Comment:
handshake times out when I'm running the test from this PR.  Just to be explicit - the only change I did is also delaying `transportFac(ch)`.
Yes, client initiates handshake on `channelActive`